packages/syft/src/syft/node/gateway.py

```diff
@@ -11,10 +11,10 @@ class Gateway(Node):
     def post_init(self) -> None:
         self.node_type = NodeType.GATEWAY
         super().post_init()
-        try:
-            self.connect_to_vpn_self()
-        except Exception as e:
-            print("Error connecting to VPN: ", e)
+        # try:
```

Comment:
@shubham3121 , for info, I have commented it out, as we are not currently using tailscale, headscale.This was causing very slow startup times, while launching a gateway.